src/fitnesse/slim/SlimExpressionEvaluator.java

```diff
@@ -34,6 +41,9 @@ public void setContext(Map<String, MethodExecutionResult> variables) {
     for (Map.Entry<String, MethodExecutionResult> entry : variables.entrySet()) {
       String key = entry.getKey();
       Object value = entry.getValue().getObject();
+      if(value.toString().startsWith("!{")) {
```

Comment:
Shouldn't the handling for 'wiki side' expressions be separate from the 'slim server side'?
For slim-server-side I don't expect to ever encounter this prefix, correct?